scripts/runqemu-extract-sdk

```diff
@@ -87,7 +87,7 @@ echo "Extracting rootfs tarball using pseudo..."
 echo "$PSEUDO $PSEUDO_OPTS tar -C \"$SDK_ROOTFS_DIR\" $TAR_OPTS \"$ROOTFS_TARBALL\""
 $PSEUDO $PSEUDO_OPTS tar -C "$SDK_ROOTFS_DIR" $TAR_OPTS "$ROOTFS_TARBALL"
 
-DIRCHECK=`ls -l "$SDK_ROOTFS_DIR" | wc -l`
+DIRCHECK=`ls -lH "$SDK_ROOTFS_DIR" | wc -l`
```

Comment:
Not sure if we want to touch this here, since it does only indirectly deal with the added script. Maybe directly contribute this to upstream?